templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -115,11 +115,13 @@ export default function ChatInput(
       onSubmit={onSubmit}
       className="rounded-xl bg-white p-4 shadow-xl space-y-4"
     >
-      <ChatResources
-        isLoading={props.isLoading}
-        resources={inputResources}
-        removeResource={removeResource}
-      />
+      {!uploadedCsv && (
```

Comment:
@thucpn hmm, this update will not show in the UI but the uploaded files still have multiple files if user add one more file, right?